test/FunctionalTests/Microsoft.OData.Edm.Tests/Validation/ValidationRulesTests.cs

```diff
@@ -803,7 +803,7 @@ public void NavigationPropertyWrongMultiplicityForDependent()
 
             EdmEntityType type2 = new EdmEntityType("ns", "type2");
             var Id2 = type2.AddStructuralProperty("Id2", EdmCoreModel.Instance.GetInt32(false));
-            
+

```

Comment:
nit: remove extra line

Reply:
this removes the extra spaces used to format the line for consistency.